gcloud/logging/_gax.py

```diff
@@ -413,16 +412,9 @@ def metric_delete(self, project, metric_name):
             raise
 
 
-def _build_paging_options(page_token=None):
-    """Helper for :meth:'_PublisherAPI.list_topics' et aliae."""
-    if page_token is None:
-        page_token = INITIAL_PAGE
-    options = {'page_token': page_token}
-    return CallOptions(**options)
-
-
 def _mon_resource_pb_to_mapping(resource_pb):
-    """Helper for  :func:_log_entry_pb_to_mapping"""
+    """Helper for  :func:_log_entry_pb_to_mapping
+    """
```

Comment:
Please no dangling `"""` and also use a backtick after `:func:` (and to close the function reference.